src/gsy_e/models/strategy/forward/load.py

```diff
@@ -33,17 +44,31 @@ def state(self) -> "LoadState":
     def event_activate(self, **kwargs):
         self._energy_params.event_activate_energy(self.area)
 
+    def remove_order(self, market: "ForwardMarketBase", market_slot: DateTime, order_uuid: str):
+        bids = [bid
+                for bid in market.slot_bid_mapping[market_slot]
+                if bid.buyer == self.owner.name and bid.id == order_uuid]
+        if not bids:
+            self.log.error("Bid with id %s does not exist on the market %s %s.",
+                           order_uuid, market.market_type, market_slot)
+        for bid in bids:
+            market.delete_bid(bid)
```

Comment:
You probably forgot to adapt here:
```suggestion
      market.delete_bid(bids[0])
```

Reply:
Good catch, thanks!